src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb

```diff
@@ -111,6 +112,146 @@ class c
             End Using
         End Function
 
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function ExplicitInvokeDateComment(showCompletionInArgumentLists As Boolean) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                <Document><![CDATA[
+using System;
+class c
+{
+    void goo()
+    {
+        // lang=date
+        var v = "$$";
```

Comment:
this is new functionality.  i moved DateTime embedded literal support to use the common 'language detection' mechanism that was added for regex/json.  So now you can also get support by adding this comment, or calling an api with the new attribute on it.